changelog.d/14959.doc

```diff
@@ -0,0 +1,2 @@
+Update delegation documentation to clarify that SRV DNS delegation does not eliminate all needs to serve files from
+.well-known locations. @williamkray
```

Comment:
CI's being a bit picky about the format of the changelog entry. Let's try this.
```suggestion
Update delegation documentation to clarify that SRV DNS delegation does not eliminate all needs to serve files from .well-known locations. Contributed by @williamkray.
```

Reply:
updated!